assets/alertmanager/service.yaml

```diff
@@ -21,7 +21,6 @@ spec:
     targetPort: tenancy
   selector:
     alertmanager: main
-    app: alertmanager
```

Comment:
Should we remove the label from the Alertmanager instance as well?

Reply:
yes, this need be removed, too

Reply:
Not sure about this comment. Which label are we talking about?

Reply:
I assume there is an `app` label on the alertmanager pods as well. If we are removing it from the selector, should we also remove it from the pods?

Reply:
Understood, in that case this should fix itself in the future.